tutorials/holoscan_debugging/debug_legacy.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+# This script builds Holoscan SDK with debug symbols and runs the Endoscopy Tool Tracking application with GDB.
+# Usage: ./debug_legacy.sh [build_type:debug,rel-debug,release] [holoscan_sdk_tag:v2.2.0] [holoscan_image:nvcr.io/nvidia/clara-holoscan/holoscan:v2.2.0-dgpu]
+
+build_type=${1:-rel-debug}
+holoscan_sdk_tag=${2:-v2.2.0}
+holoscan_image=${3:-nvcr.io/nvidia/clara-holoscan/holoscan:v2.2.0-dgpu}
+
+SCRIPT_DIR=$(dirname $(realpath $0))
+HOLOHUB_ROOT=$(realpath "${SCRIPT_DIR}/../..")
+tmp_dir=${SCRIPT_DIR}/tmp/holoscan_debugging
+mkdir -p ${tmp_dir}
+
+# Download and build Holoscan SDK
+pushd ${tmp_dir}
+if [ ! -d holoscan-sdk ]; then
+    git clone git@github.com:nvidia-holoscan/holoscan-sdk.git
+fi
+cd holoscan-sdk
+git checkout ${holoscan_sdk_tag}
+# Apply patch to fix broken Vulkan SDK source
+# https://github.com/nvidia-holoscan/holoscan-sdk/issues/30
+git reset --hard HEAD
+git apply ${SCRIPT_DIR}/holoscan_sdk_20240723_1.diff
```

Comment:
File "holoscan_sdk_20240723_1.diff" not found - forgot to include it in folder?

Reply:
Apologies, missed this on check-in. Please try again now.